modules/apps/object/object-web/src/main/resources/META-INF/resources/js/components/ObjectField/ObjectFieldFormBase.tsx

```diff
@@ -245,11 +246,13 @@ export default function ObjectFieldFormBase({
 			return true;
 		}
 
-		if (
-			oneToManyRelationship &&
-			oneToManyRelationship.deletionType !== 'disassociate'
-		) {
-			return false;
+		if (oneToManyRelationship) {
+			if (oneToManyRelationship.edge) {
+				return true;
+			}
+			else if (oneToManyRelationship.deletionType !== 'disassociate') {
+				return false;
+			}
```

Comment:
Hey @MarinhoFeliphe 

If we follow sort, the logic broken because the second conditional is `oneToManyRelationship.deletionType !== 'disassociate` and when _objectRelationship_ is edge, also is cascade (`!== 'disassociate`). So, if we follow sort, every time when _objectRelationship_ is edge will enter the first conditional that returns false.

Reply:
Hey @murilostodolni, so we need to do this:

```typescript
if (oneToManyRelationship && oneToManyRelationship.deletionType !== 'disassociate') {
  return oneToManyRelationship.edge;
}
```